tinkerpop3/src/main/java/io/shiftleft/overflowdb/storage/OdbStorage.java

```diff
@@ -12,8 +13,10 @@
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 public class OdbStorage implements AutoCloseable {
```

Comment:
IMO the readability would benefit if we move all purely index-related code into a separate unit/section that's wired in, e.g. a as a abstract class, or a separate class member. There's many ways, but the goal is: keep this class small and readable.

Reply:
Moved almost everything to OdbIndexManager